extension/src/frequency.rs

```diff
@@ -233,6 +233,47 @@ impl SpaceSavingTransState {
         }
     }
 
+    fn ingest_aggregate_data(
+        &mut self,
+        val_count: u64,
+        values: &DatumStore,
+        counts: &[u64],
+        overcounts: &[u64],
+    ) {
+        self.total_vals = val_count;
```

Comment:
Should this one assert TEXTOID?

Reply:
Also wondering this, unless there's a reason for only checking it in ingest_aggregate_ints()?

Reply:
This same function is called by both text aggregates and raw aggregates (the AnyElement version of this).  So the OID can really be anything here.